src/DynamoPackages/PackageLoader.cs

```diff
@@ -75,15 +75,17 @@ public IEnumerable<IExtension> RequestedExtensions
 
         /// <summary>
         /// Returns the default package directory where new packages will be installed
-        /// The default package directory is currently the second entry in the list
+        /// This is the first non standard library directory
```

Comment:
Won't this always be the second one?

Reply:
No, because the package search paths can be manually sorted by users to configure load order of packages.

Reply:
Depends on where in the list standard library currently are.